src/spec/sync/product-sync.spec.coffee

```diff
@@ -287,10 +287,56 @@ describe 'ProductSync', ->
           { action: 'changePrice', priceId: 'p-10', price: {value: {currencyCode: 'USD', centAmount: 1257}, country: 'US' }}
         ]
         version: oldProduct.version
+      expect(update).toEqual expected_update
 
-      # console.log JSON.stringify(update, null, 2)
+    it 'should create `changePrice` action if price selection is unique', ->
+      oldProduct =
+        id: '123'
+        version: 1
+        masterVariant:
+          id: 1
+          sku: 'v1'
+          prices: [
+            {id: 'p-1', value: {currencyCode: 'EUR', centAmount: 100}, validUntil: '2019-10-16'},
+            {id: 'p-2', value: {currencyCode: 'EUR', centAmount: 1000}, country: 'DE'},
```

Comment:
Maybe you could add also price with minimum scope also:

`{id: 'p-3', value: {currencyCode: 'EUR', centAmount: 1000}}`

And check if it gets updated also.

Reply:
Added here: https://github.com/sphereio/sphere-node-sdk/pull/251/commits/ebb0d7498514593062a76580d5fe63dbc8d054c3